terraform/modules/marklogic/monitoring_dashboard.tf

```diff
@@ -162,6 +162,26 @@ resource "aws_cloudwatch_dashboard" "main" {
           "y" : 3,
           "x" : 18,
           "type" : "metric",
+          "properties" : {
+            "metrics" : [
+              ["${var.environment}/MarkLogic", "scripted-metrics", "metric", "task-server-currently-processing", { "color" : "#e377c2", "label" : "Current processing" }],
+              ["...", "task-server-host-max-queue-size", { "color" : "#9467bd", "label" : "Queue size (host)" }],
```

Comment:
The label is queue size but the metric is max queue size? Just checking that's right

Reply:
It's the longest queue of the three task servers (one on each host). Maybe "Queue size (max of hosts)" would be better?

Reply:
Have gone with that